src/snapred/backend/data/LocalDataService.py

```diff
@@ -1328,8 +1367,9 @@ def _readLiveData(self, ws: WorkspaceName, duration: int):
             "load live-data chunk",
             OutputWorkspace=ws,
             Instrument=Config["liveData.instrument.name"],
-            AccumulationMethod=Config["liveData.accumulationMethod"],
+            AccumulationMethod="Replace",
             StartTime=startTime,
+            PreserveEvents=False,
```

Comment:
Should this be a parameter on the `_readLiveData` method itself?  I imagine this was done with reading metadata in mind.